TitoGrine/SINF_Project
# Added the warehouse map modal.

Added the warehouse modal map.

How to test:
 - Run the following commands from the project root:
         - `cd backend && npm run install && npm run start`
         - `cd frontend && npm run install && npm run start`
- Login to the site
- In the main page click the Warehouse Map button and confirm that a modal with the map appears
- Go into one of the other pages (e.g. Supplier), click the Warehouse Map button in the navbar and confirm that a modal with the map appears
- Additionally confirm that the standard modal behavior works as expected
